jekyll/_cci2/glossary.adoc

```diff
@@ -93,6 +93,10 @@ A CircleCI project shares the name of the code repository for which it automates
 
 A feature that enables building, running, and pushing images to Docker registries from within a Docker <<#executor,executor>> job. Visit the <<building-docker-images#,Runner Docker Commands>> page to learn more.
 
+[#resource-class]
+== Resource class
+A configuration option for specifying compute resource sizing for a job. Also used to identify a self-hosted runner, and configure a job to run on that self-hosted runner. Read more on the link:/docs/resource-class-overview[resource class overview] page.
```

Comment:
"also used to identify a self-hosted runner"

it is a label used to identify a pool of self-hosted runners and used to configure a job to run on a self-hosted runner from that pool.  right now it reads like it is used to identify individual self-hosted runners.